features/cellular/framework/device/CellularStateMachine.h

```diff
@@ -180,6 +180,15 @@ class CellularStateMachine {
     nsapi_event_t _current_event;
     int _status;
     PlatformMutex _mutex;
+
+    // Cellular state timeouts
+    int _state_timeout_power_on;
+    int _state_timeout_sim_pin;
+    int _state_timeout_registration;
+    int _state_timeout_network;
+    int _state_timeout_connect;
```

Comment:
could change to _state_timeout_attach to avoid confusion as state machine does not do any actual connecting anymore.